### PR TITLE
Update writing-good-workers.md

### PR DESCRIPTION
**Pull request by berndruecker**
## Description

See https://github.com/camunda/camunda-docs/issues/2446#issuecomment-1743510729 - the table was broken and is now fixed

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [X] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@berndruecker this table is still empty. Are you missing a commit with content for this table?

**Comment by akeller** [comment]
This needs to be backported.

File: versioned_docs/version-8.2/components/best-practices/development/writing-good-workers.md [hunk 1]
```diff
@@ -264,8 +264,9 @@ Node.JS code scales pretty well and there is no specific thread pool defined or
 
 This makes the recommendation very straight-forward:
 
-||Reactive code|
-| Parallelism | Event loop provided by Node.JS |
+|              | Reactive code|
+| ------------ | ------------ |
```
[the remark this answers is not in the document]
Comment (akeller) [reply versioned_docs/version-8.2/components/best-practices/development/writing-good-workers.md]: Resolve in recent commit.

**Review by pepopowitz — DISMISSED** [review]

**Comment by christinaausley** [comment]
Hi @berndruecker! Let me know if I can assist with backporting, and please sign the CLA one more time 😄

**Review by akeller — APPROVED** [review]

**Pull request merged**